h/static/scripts/controllers/form-controller.js

```diff
@@ -11,6 +11,14 @@ function shouldAutosubmit(type) {
 }
 
 /**
+ * Return true if a form field should be hidden until the user starts
+ * editing the form.
+ */
+function isHiddenField(el) {
```

Comment:
While I think `@example` is useful for public APIs, IMO adding it to all internal functions would end up making the code harder to read - because you end up adding many lines that don't actually do anything. I will add a type annotation for `el` though.